cockatrice/src/dlg_connect.cpp

```diff
@@ -155,10 +153,10 @@ void DlgConnect::rebuildComboBoxList()
 
     if (savedHostList.size() == 1)
     {
-        settingsCache->servers().addNewServer("Woogerworks Cockatrice", "cockatrice.woogerworks.com", "4747", "", "", false);
-        settingsCache->servers().addNewServer("Chickatrice Cockatrice", "chickatrice.net", "4747", "", "", false);
-        settingsCache->servers().addNewServer("Cockatric.es Cockatrice", "cockatric.es", "4747", "", "", false);
-        settingsCache->servers().addNewServer("Tetrarch Cockatrice", "mtg.tetrarch.co", "4747", "", "", false);
+        settingsCache->servers().addNewServer("Woogerworks", "cockatrice.woogerworks.com", "4747", "", "", false);
+        settingsCache->servers().addNewServer("Chickatrice", "chickatrice.net", "4747", "", "", false);
+        settingsCache->servers().addNewServer("Cockatric.es", "cockatric.es", "4747", "", "", false);
+        settingsCache->servers().addNewServer("Tetrarch", "mtg.tetrarch.co", "4747", "", "", false);
```

Comment:
Do we add `cockatrice` to every server name for a reason?
It looks like unneeded information which just clutters things?

Regarding https://github.com/Cockatrice/Cockatrice/wiki/Public-Servers
`cockatrice.es` is branded in lower case

Can we please have a link to public serves wiki in that dialog as well? In case somebody runs an older software or we don't include changes or new servers quickly enough...